### PR TITLE
✨ Show NFT model in NFT detail page and collect dialog

### PR DESCRIPTION
**Pull request by williamchong**
Requires https://github.com/likecoin/likecoin-api-public/pull/424

Appears as 3d on desktop for collect dialog
image

Only appears in mobile for NFT detail page
image

### CODE REVIEW

File: src/components/EventModal/Collect/index.vue [hunk 1]
```diff
@@ -22,6 +22,21 @@
       v-if="isCompleted"
       #message
     >
+      <client-only>
+        <model-viewer
+          v-if="nftModelURL"
+          id="book"
```
Comment (nwingt) [inline src/components/EventModal/Collect/index.vue:28]: Is this ID mean anything special?

File: src/components/EventModal/Collect/index.vue [hunk 2]
```diff
@@ -22,6 +22,21 @@
       v-if="isCompleted"
       #message
     >
+      <client-only>
+        <model-viewer
```
Comment (nwingt) [inline src/components/EventModal/Collect/index.vue:26]: should add a margin top for this element

File: src/pages/nft/class/_classId/_nftId.vue [hunk 3]
```diff
@@ -178,8 +178,45 @@
...
+                  'w-full',
+                  'flex',
+                  'flex-col',
+
+                  'overflow-hidden',
+
+                  'border-[1px]',
+                  'border-shade-gray',
+                ]"
+                :has-padding="false"
```
Comment (nwingt) [inline src/pages/nft/class/_classId/_nftId.vue:196]:
```suggestion
```

File: src/components/EventModal/Collect/index.vue (hunk 2 above)
Comment (williamchong) [reply src/components/EventModal/Collect/index.vue:26]: how much do you suggest?

File: src/components/EventModal/Collect/index.vue (hunk 1 above)
Comment (williamchong) [reply src/components/EventModal/Collect/index.vue:28]: not in vue context, will remove

File: src/components/EventModal/Collect/index.vue (hunk 2 above)
Comment (williamchong) [reply src/components/EventModal/Collect/index.vue:26]: image
effect of 12px

Comment (nwingt) [reply src/components/EventModal/Collect/index.vue:26]: Seems ok

**Review by nwingt — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**